src/main/kotlin/org/polystat/j2eo/treeMapper/TreeMappings.kt

```diff
@@ -4,7 +4,138 @@ package org.polystat.j2eo.treeMapper
+import JavaParser.LiteralExpressionContext
+import JavaParser.ParenthesizedExpressionContext
+import JavaParser.MethodCallContext
+import JavaParser.ArgumentsContext
+import JavaParser.InnerCreatorContext
+import JavaParser.PrimaryExpresionContext
+import JavaParser.MethodInvocationContext
+import JavaParser.InstanceCreationContext
+import JavaParser.InstanceOfExpressionContext
+import JavaParser.CastExpresionContext
+import JavaParser.ArrayAccessContext
+import JavaParser.PostfixExpressionContext
+import JavaParser.PrefixExpressionContext
+import JavaParser.GT
+import JavaParser.LT
+import JavaParser.BinaryExpressionContext
+import JavaParser.URSHIFT_ASSIGN
+import JavaParser.MOD_ASSIGN
+import JavaParser.TILDE
+import JavaParser.BANG
+import JavaParser.VAR
+import JavaParser.DEC
+import JavaParser.INC
+import JavaParser.QUESTION
+import JavaParser.INSTANCEOF
+import JavaParser.DOT
+import JavaParser.RSHIFT_ASSIGN
+import JavaParser.LSHIFT_ASSIGN
+import JavaParser.XOR_ASSIGN
+import JavaParser.MOD
+import JavaParser.NOTEQUAL
+import JavaParser.AND_ASSIGN
+import JavaParser.LE
+import JavaParser.GE
+import JavaParser.CARET
+import JavaParser.BITOR
+import JavaParser.OR_ASSIGN
+import JavaParser.BITAND
+import JavaParser.AND
+import JavaParser.OR
+import JavaParser.EQUAL
+import JavaParser.ASSIGN
+import JavaParser.DIV_ASSIGN
+import JavaParser.SUB_ASSIGN
+import JavaParser.MUL_ASSIGN
+import JavaParser.ADD_ASSIGN
+import JavaParser.DIV
+import JavaParser.SUB
+import JavaParser.MUL
+import JavaParser.ADD
+import JavaParser.SwitchLabelContext
+import JavaParser.SwitchBlockStatementGroupContext
+import JavaParser.StatementTryBlockContext
+import JavaParser.StatementTryResourceSpecificationContext
+import JavaParser.StatementSwitchExpressionContext
+import JavaParser.StatementSwitchContext
+import JavaParser.StatementSemiContext
+import JavaParser.StatementIdentifierLabelContext
+import JavaParser.StatementReturnContext
+import JavaParser.StatementDoContext
+import JavaParser.StatementBlockLabelContext
+import JavaParser.StatementForContext
+import JavaParser.StatementContinueContext
+import JavaParser.StatementBreakContext
+import JavaParser.StatementAssertContext
+import JavaParser.StatementExpressionContext
+import JavaParser.StatementIfContext
+import JavaParser.StatementWhileContext
+import JavaParser.StatementContext
+import JavaParser.ConstructorDeclarationContext
+import JavaParser.FieldDeclarationContext
+import JavaParser.PUBLIC
+import JavaParser.PRIVATE
+import JavaParser.STATIC
+import JavaParser.PROTECTED
+import JavaParser.ABSTRACT
+import JavaParser.FINAL
+import JavaParser.STRICTFP
+import JavaParser.ClassOrInterfaceModifierContext
+import JavaParser.MethodDeclarationContext
+import JavaParser.MethodBodyContext
+import JavaParser.FormalParametersContext
+import JavaParser.FormalParameterListContext
+import JavaParser.BlockContext
+import JavaParser.FormalParameterContext
+import JavaParser.BlockStatementContext
 import tree.*
```

Comment:
@includealex we still have wildcard imports

Reply:
I can see suppression of the warning below: `ktlint-disable`.
Can we enable it?

Reply:
@mximp yes, we can